### PR TITLE
Refactored code of Awesome book project

### PR DESCRIPTION
**Pull request by abenaTenkorang**
### Awesome Book refracturing with ES6
In this pull request, I update JavaScript file by using ES6 syntax.

Changes Made

- I grouped Classes and functions into modules
- Create time and date container in HTML to display time and date
-  I changed all functions to arrow functions
-  Fixing all Linter Errors

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,17 @@
...
\ No newline at end of file
+import './modules/button_functions.js';
+import DateTimeFormat from './modules/dateTime.js';
+import { addBooks } from './modules/add_remove.js';
+import { CheckLocalStorage, booksArr } from './modules/onload.js';
+import './modules/extrafunctions.js';
+
+const displayTime = () => setInterval(() => {
+  document.querySelector('#date').innerHTML = DateTimeFormat();
+}, 1000);
```
Comment (Qoosim) [inline index.js:9]: - [ ] The position of the date on the page is not properly positioned. Kindly fix that.

[app-date.webm](https://user-images.githubusercontent.com/26597432/206987565-aef5f3fd-f9bd-467d-842a-b279d8510faf.webm)

File: modules/add_remove.js [hunk 2]
```diff
@@ -1 +1,25 @@
...
+    author.value = '';
+    title.focus();
+  }
+};
+
+export const deleteBook = (el) => {
+  if (el.classList.contains('remove-books')) {
+    el.parentElement.remove();
+  }
+};
```
Comment (Qoosim) [inline modules/add_remove.js:49]: - [ ] Deleted book still re-display on the browser after page reloading. Kindly look into that and fix.

[app-del.webm](https://user-images.githubusercontent.com/26597432/206988343-5f98bd5a-095b-4d7e-a6d4-0af1b1e74823.webm)

File: modules/extrafunctions.js [hunk 3]
```diff
@@ -1 +1,38 @@
...
+      title: title.value,
+      author: author.value,
+    };
+    booksArr.push(theBook);
+    addBooks(booksArr);
+    localStorage.setItem('booksArr', JSON.stringify(booksArr));
+    document.querySelector('.emptymsg').style.display = 'none';
+    booklist.style.border = '1px solid black';
+  }
+});
```
Comment (Qoosim) [inline modules/extrafunctions.js]: - [ ] New book object created override the previously added book object. At this moment, you can only add one book object to the app. Kindly look into what the issue is and fix.

[app-behavior.webm](https://user-images.githubusercontent.com/26597432/206988832-1305158c-e7a3-4996-8133-ddc91ff8fc0c.webm)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @abenaTenkorang, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Your README file is empty. Kindly use this [new template](https://github.com/microverseinc/readme-template) to update your README file.

File: index.js (hunk 1 above)
Comment (abenaTenkorang) [reply index.js:9]: Position fixed. Thank you

File: modules/add_remove.js (hunk 2 above)
Comment (abenaTenkorang) [reply modules/add_remove.js:49]: Delete books no longer re-appears on the browser after display.

File: modules/extrafunctions.js (hunk 3 above)
Comment (abenaTenkorang) [reply modules/extrafunctions.js]: New book adds on to previous books.

**Review by danmainah — APPROVED** [review]
Hi @abenaTenkorang , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

- [x] Kindly ensure the future branch readme file is professional, add content using the microverse readme template.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**